.storybook/options.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+// If this env variable is set, we won't fork the type checker.
+const DONT_FORK_TS_CHECKER = 'DONT_FORK_TS_CHECKER'
```

Comment:
Do we know that we are going to use this environment variable, or is it here just in case? If the latter, then I think we should remove the environment variable and add it in the future if we find out that we really need it.

Reply:
Sweet, I only added it because I thought you wanted an explicit way of changing back. More than happy to remove it :smile: